__tests__/app.tests.js

```diff
@@ -111,4 +110,71 @@ describe("get user data from database", () => {
     });
 });
 
+//requests alters votes on db and returns altered review
+describe("PATCH specific review to increase votes", () => {
+	test("POST /api/reviews/1 returns data required", () => {
+    const votes = { inc_votes: 10};
+    return request(app)
+        .post("/api/reviews/1")
```

Comment:
We're doing a .patch request here not a post.
A post would create a new entry where we want to edit an existing one.